mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -1583,7 +1583,8 @@ static void processPassthroughAttrs(llvm::Function *func, LLVMFuncOp funcOp) {
     // explicit attribute.
     // Also skip the vscale_range, it is also an explicit attribute.
```

Comment:
Nit: Can you adapt the comment to be either more general or explicitly list this newly added attibute?

Reply:
I've reworked this to a `ExplicitAttributes` list rather than this ever-growing 'or' expression :+1: